redash/query_runner/treasuredata.py

```diff
@@ -102,16 +102,13 @@ def run_query(self, query, user):
         cursor = connection.cursor()
         try:
             cursor.execute(query)
-            columns_data = [(row[0], cursor.show_job()['hive_result_schema'][i][1]) for i,row in enumerate(cursor.description)]
-
-            columns = [{'name': col[0],
-                'friendly_name': col[0],
-                'type': TD_TYPES_MAPPING.get(col[1], None)} for col in columns_data]
+            columns_tuples = [(i[0], TD_TYPES_MAPPING.get(i[1], None)) for i in cursor.show_job()['hive_result_schema']]
```

Comment:
Previous implementation was using `cursor.description` while current one is just using `cursor.show_job()`. I'm not familiar enough with TD to know if this is supported across all versions. Can you explain the difference?

Reply:
Yes, both call the same Treasure Data API request in td-client. `hive_result_schema` included in this response is an array of column name and type.

The difference between the two is that `cursor.show_job()` returns  `hive_result_schema` and other responce directly, but `cursor.description` retrieves and returns only the column names from `hive_result_schema`.

Previous implementation got column names from `cursor.description` and got types from `cursor.show_job()`, but I thought that using `cursor.show_job()` only is simpler.

Because the implementation of `cursor.show_job()` has not been changed since the DB API was introduced to td-client, it should work fine for all versions.